packages/base-driver/lib/basedriver/helpers.js

```diff
@@ -409,6 +409,20 @@ export function generateDriverLogPrefix(obj, sessionId = null) {
   return `${obj.constructor.name}@${node.getObjectId(obj).substring(0, 4)}`;
 }
 
+/**
+ * Register the session in the context to be used in the logs
+ *
+ * @param {string} sessionId session identifier
+ */
+export function registerLogContext(sessionId) {
+  const contextStorage = global._global_npmlog?.asyncStorage;
```

Comment:
each driver has its own log instance. We may simply unwrap it and get the storage instance from there

Reply:
which means it makes sense to make this to an instance method rather than a helper

Reply:
I moved it to AppiumLogger.